core/src/main/java/psiprobe/tools/AccessorFactory.java

```diff
@@ -43,7 +43,13 @@ public static Accessor getInstance() {
   private static Accessor getReflective() {
     try {
       return new ReflectiveAccessor();
-    } catch (Exception ex) {
+    } catch (ClassNotFoundException e) {
```

Comment:
At the moment, we are still compiling to java 6.  Do you know if multi-catch is something we can code and compile towards 6?  We haven't bumped up fully to 7 yet.